pattern-categorization/innersource-program-mind-map.md

```diff
@@ -38,6 +38,10 @@
 
 ##### [Cross-Team Project Valuation](https://patterns.innersourcecommons.org/p/crossteam-project-valuation)
 
+#### How to determine a project's release process and cadence
```

Comment:
This text here should describe the "specific problem occurring in an InnerSource context.
See [reference](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/pattern-categorization/README.md#innersource-program-mind-map).

I am wondering if we could make it more specifically about InnerSource?
i.e. why is the release process/cadence relevant for consuming teams in an InnerSource collaboration approach?

Reply:
Ok, I see the difference. How about something like, 

"How to determine a project's commitment to releases and transparency"

or

"How to measure a project's commitment to frequent releases and documentation"

Let me know if either of those get at the problem category in a better way. Happy to tweak before committing something again, easier to go back and forth in comments 🙂

Reply:
thanks @dgrizzanti 

I would try to work more with the problem statement that you have in your pattern, and less with the specific solution.

Brainstorming some ideas:

> Assessing project maturity

Probably too generic.

> Assessing longevity and release stability of a project

This tries to paraphrase what you have in the [problem](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/667272a8668174542e59f695660b5fd863b9e9f6/patterns/2-structured/release-process.md#problem) section.
How to say this clearly but also very short?

> Can we trust the project for an extended period?

Similar to the one above but using the words from your problem statement.

I think I like this last one best but am certainly not committed to it, if you have a better idea :)

What I like about this last one is that one could even imagine that for that problem statement there could be multiple patterns that help with it. e.g. "Standard Base Documentation" might also increase trust in the project.

_Aside:_
Just double checking that you also saw [my feedback about the Context section](https://github.com/InnerSourceCommons/InnerSourcePatterns/pull/524#pullrequestreview-1303658556) of this pattern?

Reply:
@spier thanks for the reminder - I saw this but then it got buried in the comments. I've updated the problem category based on your suggestions and also added more details to `Context` and `Forces`. Let me know what you think when you have some time. Thanks!